export default {
  '10.1101/2020.03.22.002386': '<p id="p-2">An outbreak of the novel coronavirus SARS-CoV-2, the causative agent of COVID-19 respiratory disease, has infected over 290,000 people since the end of 2019, killed over 12,000, and caused worldwide social and economic disruption<sup><a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>,<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a></sup>. There are currently no antiviral drugs with proven efficacy nor are there vaccines for its prevention. Unfortunately, the scientific community has little knowledge of the molecular details of SARS-CoV-2 infection. To illuminate this, we cloned, tagged and expressed 26 of the 29 viral proteins in human cells and identified the human proteins physically associated with each using affinity-purification mass spectrometry (AP-MS), which identified 332 high confidence SARS-CoV-2-human protein-protein interactions (PPIs). Among these, we identify 67 druggable human proteins or host factors targeted by 69 existing FDA-approved drugs, drugs in clinical trials and/or preclinical compounds, that we are currently evaluating for efficacy in live SARS-CoV-2 infection assays. The identification of host dependency factors mediating virus infection may provide key insights into effective molecular targets for developing broadly acting antiviral therapeutics against SARS-CoV-2 and other deadly coronavirus strains.</p>',
  '10.1101/2020.02.26.966986': '<p id="p-2">Swarming is form of collective bacterial motion enabled by flagella on the surface of semi-solid media<sup><a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a></sup>. Many bacterial species exhibit non-genetic adaptive resistance to a broad range of antibiotics only when swarming (SR)<sup><a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>-<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a></sup>. While the swarming population as a whole survives antibiotic challenge, it nonetheless sustains considerable cell death<sup><a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a></sup>. In this study focused mainly on <em>E. coli</em>, an initial analysis of antibiotic-induced killing patterns of swarm vs planktonic cells indicated that death of a sub-population is beneficial to the swarm in promoting SR. Introduction of pre-killed cells into a swarm indeed enhanced SR, allowing us to purify the SR factor from killed cells, and to establish that cell death is directly involved in SR. The SR-enhancing factor was identified to be AcrA, a periplasmic component of a tripartite RND efflux pump<sup><a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a></sup>, of which the outer membrane component TolC is shared by multiple drug efflux pumps<sup><a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a></sup>. We show that AcrA stimulates drug efflux in live cells by interacting with TolC from the outside. This stimulus acts as a signal to activate efflux in the short term, and to induce the expression of other classes of efflux pumps in the long term, amplifying the response and establishing SR. We call this phenomenon ‘necrosignaling’, and show the existence of species-specific necrosignaling in both Gram-positive and Gram-negative bacteria. Our results have implications for the adaptive resistance of other surface-associated bacterial collectives such as biofilms. Given that such resistance is known to be an incubator for evolving genetic resistance<sup><a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a></sup>, our findings may also be relevant to chemotherapy-resistant cancers.</p>',
  '10.1101/2020.01.22.915660': '<p id="p-2">Over the past 20 years, several coronaviruses have crossed the species barrier into humans, causing outbreaks of severe, and often fatal, respiratory illness. Since SARS- CoV was first identified in animal markets, global viromics projects have discovered thousands of coronavirus sequences in diverse animals and geographic regions. Unfortunately, there are few tools available to functionally test these novel viruses for their ability to infect humans, which has severely hampered efforts to predict the next zoonotic viral outbreak. Here we developed an approach to rapidly screen lineage B betacoronaviruses, such as SARS-CoV and the recent 2019-nCoV, for receptor usage and their ability to infect cell types from different species. We show that host protease processing during viral entry is a significant barrier for several lineage B viruses and that bypassing this barrier allows several lineage B viruses to enter human cells through an unknown receptor. We also demonstrate how different lineage B viruses can recombine to gain entry into human cells and confirm that human ACE2 is the receptor for the recently emerging 2019-nCoV.</p>',
  '10.1101/861799': '<p id="p-3">Antibiotics are losing efficacy due to the rapid evolution and spread of resistance. Treatments targeting bacterial virulence factors have been considered as alternatives because they target virulence instead of pathogen viability, and should therefore exert weaker selection for resistance than conventional antibiotics. However, antivirulence treatments rarely clear infections, which compromises their clinical applications. Here, we explore the potential of combining antivirulence drugs with antibiotics against the opportunistic human pathogen <em>Pseudomonas aeruginosa</em>. We combined two antivirulence compounds (gallium, a siderophore-quencher, and furanone C-30, a quorum sensing-inhibitor) together with four clinically relevant antibiotics (ciprofloxacin, colistin, meropenem, tobramycin) in 9×9 drug concentration matrices. We found that drug-interaction patterns were concentration dependent, with promising levels of synergies occurring at intermediate drug concentrations for certain drug pairs. We then tested whether antivirulence compounds are potent adjuvants, especially when treating antibiotic resistant clones. We found that the addition of antivirulence compounds to antibiotics could restore growth inhibition for most antibiotic resistant clones, and even abrogate or reverse selection for resistance in five drug combination cases. Molecular analyses suggest that selection against resistant clones occurs when resistance mechanisms involve restoration of protein synthesis, but not when efflux pumps are upregulated. Altogether, our work provides a first systematic analysis of antivirulence-antibiotic combinatorial treatments and suggests that such combinations have a high potential to be both effective in treating infections and in limiting the spread of antibiotic resistance.</p>',
  '10.1101/862755': '<p id="p-5"><em>Streptococcus pneumoniae</em> is a genetically diverse human-adapted pathogen commonly carried asymptomatically in the nasopharynx. We have recently shown that a single nucleotide polymorphism (SNP) in the raffinose pathway regulatory gene <em>rafR</em> accounts for a significant difference in the capacity of clonally-related strains to cause localised versus systemic infection. Here we have used dual RNA-seq to show that this SNP extensively impacts both bacterial and host transcriptomes in infected lungs. It affects expression of bacterial genes encoding multiple sugar transporters, and fine-tunes carbohydrate metabolism, along with extensive rewiring of host transcriptional responses to infection, particularly expression of genes encoding cytokine and chemokine ligands and receptors. The dual RNA-seq data predicted a crucial role for differential neutrophil recruitment in the distinct virulence profiles of the infecting strains and single cell analysis revealed that while reduced expression of the RafR regulon driven by a single <em>rafR</em> SNP provides a clear advantage for pneumococci to colonize the ear, in the lung it leads to massive recruitment of neutrophils and bacterial clearance. Importantly, the observed disease outcomes were confirmed by <em>in vivo</em> neutrophil depletion showing that early detection of bacteria by the host in the lung environment is crucial for effective clearance. Thus, dual RNA-seq provides a powerful tool for understanding complex host-pathogen interactions and revealed how a single bacterial SNP can drive differential disease outcomes.</p>',
  '10.1101/845081': '<p id="p-2">Several lipids of the pathogen <em>Mycobacterium tuberculosis</em> are known to promote virulence at various stages of disease. However, the inability to probe these lipids during in vivo infection makes elucidation of their pathogenic mechanisms difficult. Using chemical extraction and reconstitution methods, we were able to define the lipid composition of the outer mycomembrane of <em>Mycobacterium marinum</em> prior to infection. Combining this approach with the synthesis of clickable, semi-synthetic lipids, we introduced a chemically tractable, biologically active variant of the virulence lipid phthiocerol dimycocerosate (PDIM) into the mycomembrane. We find that following infection of zebrafish larvae, PDIM spreads away from bacterial surfaces into the membranes of both macrophage and epithelial cells that it contacts. This spreading facilitates PDIM’s role in preventing bacterium-detrimental immune activation at the site of infection.</p>',
  '10.1101/823989': '<p id="p-2">The discovery of new viruses currently outpaces our capacity for experimental examination of infection biology. To better couple virus discovery with immunology, we genetically modified zebrafish to visually report on virus infections. After generating a strain that expresses green fluorescent protein (GFP) under an interferon-stimulated gene promoter, we repeatedly observed transgenic larvae spontaneously expressing GFP days after hatching. RNA sequencing of GFP-positive zebrafish revealed a picornavirus distantly related to known viruses. We detected this virus in publicly available sequencing data from seemingly asymptomatic zebrafish in research institutes world-wide. These data revealed widespread tissue tropism and identified the clonal CG2 strain as notably vulnerable to picornavirus infection. Infection induces the expression of numerous antiviral immune defense genes in zebrafish, an effect also evident in public data. Our study describes a prevalent picornavirus that infects zebrafish, and provides new strategies for simultaneously discovering viruses and their impact on vertebrate hosts.</p>',
  '10.1101/380238': '<p id="p-3">Tumor extracellular vesicles (tumor EVs) mediate the communication between tumor and stromal cells mostly to the benefit of tumor progression. Notably, tumor EVs have been reported to travel in the blood circulation, reach specific distant organs and locally modify the microenvironment. However, visualizing these events <em>in vivo</em> still faces major hurdles. Here, we show a new method for tracking individual circulating tumor EVs in a living organism: we combine novel, bright and specific fluorescent membrane probes, MemBright, with the transparent zebrafish embryo as an animal model. We provide the first description of tumor EVs’ hemodynamic behavior and document their arrest before internalization. Using transgenic lines, we show that circulating tumor EVs are uptaken by endothelial cells and blood patrolling macrophages, but not by leukocytes, and subsequently stored in acidic degradative compartments. Finally, we prove that the MemBright can be used to follow naturally released tumor EVs <em>in vivo</em>. Overall, our study demonstrates the usefulness and prospects of zebrafish embryo to track tumor EVs <em>in vivo</em>.</p>',
} as Record<string, string|undefined>;
